code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -103,20 +103,9 @@ emp_act
 		return 1
 	if(wear_suit)
 		if(wear_suit.IsShield() && (prob(50)))

```

Comment:
move this probability to a var on the suit or just into the hit_reaction proc of each subtype